app/teachingCall/teachingCallStatus/services/teachingCallStatusService.js

```diff
@@ -15,6 +15,9 @@ class TeachingCallStatusService {
 			},
 			removeInstructorFromTeachingCall: function (teachingCallReceiptId) {
 				return ApiService.delete("/api/teachingCallView/teachingCallReceipts/" + teachingCallReceiptId);
+			},
+			getCourses: function (workgroupId, year) {
+				return ApiService.get("/api/courseView/workgroups/" + workgroupId + "/years/" + year);
```

Comment:
Does CourseService have something we could re-use instead of adding a new function?